core/src/main/scala/akka/kafka/KafkaConsumerActor.scala

```diff
@@ -140,6 +141,32 @@ private[kafka] class KafkaConsumerActor[K, V](settings: ConsumerSettings[K, V])
         self ! delayedPollMsg
       }
 
+    case SubscribeWithStartTimestamp(timestamp, topics, listener) =>
+      scheduleFirstPollTask()
+      consumer.subscribe(topics.toList.asJava, new WrappedAutoPausedListener(consumer, listener))
+      // -- Seek offset
+      consumer.poll(0)
+      val assignedTopicPartitions = consumer.assignment()
+
+      // Get offset for a given timestamp for each topic/partition couple
+      val timestampsToSearch = assignedTopicPartitions.asScala.map(
```

Comment:
Would spelling out the type of `timestampsToSearch` remove the need for that comment?